fe/fe-core/src/main/java/com/starrocks/catalog/Column.java

```diff
@@ -228,6 +228,15 @@ public Column(Column column) {
         Preconditions.checkArgument(this.type.isComplexType() ||
                 this.type.getPrimitiveType() != PrimitiveType.INVALID_TYPE);
         this.uniqueId = column.getUniqueId();
+        this.generatedColumnExpr = column.generatedColumnExpr();
+    }
+
+    public Column deepCopy() {
```

Comment:
From a design point of view, the copy constructor itself should be a DeepCopy. If there is a problem with the current implementation, should we modify the copy constructor directly instead of adding another function?

Reply:
> he copy constructor itself should be a DeepCopy

In fact, the `type` is not deeply copied in the copy constructor, but I am not sure if this is by design. That is why I add a new function.